src/reducers/rootReducer.js

```diff
@@ -32,6 +33,7 @@ const rootReducer = (history) => combineReducers({
   resources,
   toolDrawer,
   urlTracker,
+  apiModal,
```

Comment:
in support of clean, consistent code, can we keep the import and combineReducers lists in alphabetical order? thanks